markdown_writer.py

```diff
@@ -185,15 +187,19 @@ def write_to_markdown(
             # Replace any whitespace
             issue.title = issue.title.strip()
 
+            endpoint = ghe.removeprefix("https://") if ghe else "github.com"
```

Comment:
Same suggestion from [here](https://github.com/github/contributors/pull/197#discussion_r1819987751).  It might make the changes below more complex.  wdyt?

Reply:
Hello @jmeridth,
I might be confusing something in this case :)
My idea was that the replace done next and also in some other parts of the code github.com is hardcoded, but we might need to consider that the `ghe` variable is something like `https://yourgithubenterprise.com`.
In this case should't all the references to `github.com` be switched to the `yourgithubenterprise.com` domain?
Again I might be missing something in this case since I couldn't fully test the action against our GHE instance
Basically the main idea here is to use the ghe domain if the user has set it or github.com by default.
Thanks

Reply:
This ask isn't enough to block. I understand your code change. I'll do a follow-up PR to express my change.

Reply:
@jmeridth did some changes also [here](https://github.com/github/contributors/pull/197#discussion_r1820426913).
I will take a look also in this PR